### PR TITLE
DApp3 Bug Fixes

### PR DESCRIPTION
**Pull request by shahthepro**
Should fix the following from #2398 
- [x] When I go back from the phone attestation form, it closes and then separately reopens the "add verifications" screen. It should act like a navigation stack where I go directly back from phone to all attestations.
- [x] If I cancel the blockchain transaction that results from "Sign to publish", the modal prompt ("...will now ask you to sign...") won't go away and I'm stuck.
- [ ] ~~Both nav drawers should slide open instead of just appearing~~
- [x] The "Earn OGN" button should cause the right drawer to close before (or while) navigating. Otherwise, it remains open when navigating backward.
- [x] The container for the "browse" categories should be full-bleed with no margin so that there isn't some whitespace on the right and left when an icon is cut off in overflow.
- [x] `DownloadApp` should never be shown when the user is in our app.
- [x] Search view with featured categories remains open after clicking a category icon. This causes an odd UX if I navigate backward. It seems to hide after navigating backward if I wait long enough but I don't think it should be there at all.
- [x] Modals dealing with in-progress blockchain transactions should not be dismissible, particularly by clicking the screen outside of the modal.

### CODE REVIEW

File: dapps/marketplace/src/components/Dropdown.js [hunk 1]
```diff
@@ -49,10 +49,23 @@ class Dropdown extends Component {
         onMouseOut={() => (this.mouseOver = false)}
       >
         {this.props.children}
-        {this.props.content && this.props.open ? this.props.content : null}
```
Comment (nick) [inline dapps/marketplace/src/components/Dropdown.js]: This is a fairly major change as now all dropdown content will be rendered and hidden with CSS regardless if the dropdown is open... in a lot of cases that will mean loading extra data that was not required before. Why was this change needed?

File: packages/graphql/src/configs/localhost.js [hunk 2]
```diff
@@ -14,14 +14,15 @@ const config = {
   ipfsGateway: `http://${HOST}:8080`,
   ipfsRPC: `http://${HOST}:5002`,
   relayer: `http://${HOST}:5100`,
-  bridge: 'https://bridge.dev.originprotocol.com',
+  bridge: `http://${HOST}:5000`,
```
Comment (nick) [inline packages/graphql/src/configs/localhost.js]: Can we revert this, so a local bridge is not required for local dev?

Comment (shahthepro) [reply packages/graphql/src/configs/localhost.js]: Will revert it.

File: dapps/marketplace/src/components/Dropdown.js (hunk 1 above)
Comment (shahthepro) [reply dapps/marketplace/src/components/Dropdown.js]: I added CSS animation for slide in/out for `.dropdown-menu`. The animations didn't happen if the element was rendered instantly with the class `.show`. 

Didn't think about it this way. Will change it.

File: dapps/marketplace/src/components/Dropdown.js [hunk 3]
```diff
@@ -49,10 +49,23 @@ class Dropdown extends Component {
...
+        {this.props.content && this.state.open ? this.props.content : null}
       </El>
     )
   }
 }
 
 export default Dropdown
+
```
Comment (nick) [inline dapps/marketplace/src/components/Dropdown.js]: Should we remove this for now?

File: dapps/marketplace/src/pages/nav/Mobile.js [hunk 4]
```diff
@@ -49,7 +49,7 @@ const MobileNav = ({ open, onClose, onOpen }) => {
       content={
         <>
           <div className="dropdown-menu-bg" onClick={() => onClose()} />
-          <div className="dropdown-menu show">
+          <div className={`dropdown-menu${open ? ' show' : ''}`}>
```
Comment (nick) [inline dapps/marketplace/src/pages/nav/Mobile.js]: should we revert changes in this file?

File: dapps/marketplace/src/pages/nav/Profile.js [hunk 5]
```diff
@@ -173,7 +199,11 @@ const ProfileDropdownRaw = ({
   return (
     <>
       <div className="dropdown-menu-bg" onClick={onClose} />
-      <div className="dropdown-menu dropdown-menu-right show profile">
```
Comment (nick) [inline dapps/marketplace/src/pages/nav/Profile.js]: revert?

File: packages/graphql/src/configs/localhost.js [hunk 6]
```diff
@@ -21,7 +21,7 @@ const config = {
   performanceMode: localStorageHas('performanceMode'),
   graphql: `http://${HOST}:4007`,
   automine: 2000,
-  attestationIssuer: '0x5be37555816d258f5e316e0f84D59335DB2400B2',
+  attestationIssuer: '0x627306090abaB3A6e1400e9345bC60c78a8BEf57',
```
Comment (nick) [inline packages/graphql/src/configs/localhost.js]: revert?

File: dapps/marketplace/src/pages/nav/Profile.js (hunk 5 above)
Comment (shahthepro) [reply dapps/marketplace/src/pages/nav/Profile.js]: Done
